1001-2000/LIT1297Dersan.xml

```diff
@@ -39,64 +39,139 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
            <language ident="en">English</language>
            <language ident="gez">Gǝʿǝz</language>
          </langUsage>
-        <abstract>
-<p>Dǝrsāna Rufāʾel is a composition for the feast of Archangel Raphael on the 3rd of Ṗagʷǝmen.
-It consists of a homily by John Chrysostom on Raphael, a collection of miracles and two homilies on the Living Creatures
-(by Rǝtuʿa Haymānot and Chrystostom).</p>
-        </abstract>
+         <abstract>
+            <p>
+               This record identifies a collection of texts for the feast of the archangel Raphael, to be read on the 3rd of <foreign xml:lang="gez">Ṗāgʷǝmen</foreign>.
+               It consists of a <ref type="work" corresp="LIT7214HomilyJCRuf">Homily by John Chrysostom</ref>, two collections of miracles 
+               (the unity of the two collections is still to be investigated), two additional texts
+               (respectively entitled <ref type="work" corresp="LIT7215HistoryRuf">History of the consecration of the church of Rufāʾel on an island near Alexandria</ref> 
```

Comment:
If the angel is called Raphael above, you should perhaps better use this form here as well.